zhaquirks/xiaomi/aqara/motion_ac02.py

```diff
@@ -0,0 +1,111 @@
+"""Quirk for LUMI lumi.motion.ac02."""
+import logging
+from typing import Any
+
+from zigpy.profiles import zha
+from zigpy.quirks import CustomDevice
+import zigpy.types as types
+from zigpy.zcl.clusters.general import Basic, Identify, Ota, PowerConfiguration
+
+from zhaquirks import Bus
+from zhaquirks.const import (
+    DEVICE_TYPE,
+    ENDPOINTS,
+    INPUT_CLUSTERS,
+    MODELS_INFO,
+    OUTPUT_CLUSTERS,
+    PROFILE_ID,
+)
+from zhaquirks.xiaomi import (
+    IlluminanceMeasurementCluster,
+    MotionCluster,
+    OccupancyCluster,
+    XiaomiAqaraE1Cluster,
+    XiaomiPowerConfiguration,
+)
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class OppleCluster(XiaomiAqaraE1Cluster):
+    """Opple cluster."""
+
+    ep_attribute = "opple_cluster"
+    attributes = {
+        0x0102: ("detection_interval", types.uint8_t, True),
+        0x010C: ("motion_sensitivity", types.uint8_t, True),
+    }
+
+    def _update_attribute(self, attrid: int, value: Any) -> None:
+        super()._update_attribute(attrid, value)
+        if attrid == 274:
+            value = value - 65536
+            self.endpoint.illuminance.illuminance_reported(value)
+            self.endpoint.occupancy.update_attribute(0, 1)
+
+    async def write_attributes(
+        self, attributes: dict[str | int, Any], manufacturer: int | None = None
```

Comment:
Home Assistant Core 2022.5.4/Home Assistant OS 7.6 ship with Python 3.9 but https://peps.python.org/pep-0604/ is only supported with Python 3.10.
For anyone wanting to try this out, change this line to:
```python
self, attributes: dict[Union[str, int], Any], manufacturer: Union[int, None] = None
```